### PR TITLE
Update encounter_but_green for https://github.com/corona-warn-app/cwa…

### PR DESCRIPTION
**Pull request by MarlisFriedl**
…-website/issues/307

### CODE REVIEW

File: src/data/faq_de.json [hunk 1]
```diff
@@ -76,7 +76,7 @@
...
...dazu passenden Zufallscodes geschätzt, wie lange die Begegnungen jenes Tags insgesamt gedauert haben und wie nahe die Smartphones sich dabei im Durchschnitt waren. Die Entfernung wird aus der gemessenen Abschwächung des Bluetooth-Signals errechnet, die in dB (Dezibel) angegeben wird. Jedem dB-Wert lässt sich eine Entfernung im Freiraum (d.h. ohne Hindernisse im Signalweg) zuordnen. Alle Begegnungen zu einer Positivkennung, die insgesamt weniger als 10 Minuten gedauert haben (egal, wie nahe sich die Smartphones dabei gekommen sind) oder bei denen die Smartphones im Durchschnitt mehr als ca. 8 Meter Freiraum (73 dB Dämpfung) voneinander entfernt waren (egal, wie lange sie insgesamt gedauert haben), werden als unbedenklich bewertet. Deswegen kann die Corona-Warn-App Ihnen Begegnungen gemeldet haben, ohne dass die Risikobewertung sich ändert.", "Weitere Details zur Risikoermittlung mit einer Beispielrechnung finden Sie unter <a href='https://github.com/corona-warn-app/cwa-documentation/blob/master/translations/cwa-risk-assessment.de.md' target='_blank' rel='noopener noreferrer'>https://github.com/corona-warn-app/cwa-documentation/blob/master/translations/cwa-risk-assessment.de.md</a>."
```
Comment (daimpi) [inline src/data/faq_de.json:79]: the English version states: 
>This is why the Corona-Warn-App can display encounters, but the risk status stays the same.

while the German one says:
>Deswegen kann die Corona-Warn-App Ihnen Begegnungen gemeldet haben, ohne dass die Risikobewertung sich ändert.

display encounters != Begegnung gemeldet

maybe the German version could be brought in line with the English one? Something like "Deswegen kann die Corona-Warn-App Ihnen Begegnungen anzeigen, […]"?

It's not that important, just thought I'd mention it 🙂

Comment (tkowark) [reply src/data/faq_de.json]: Thanks a lot. I've changed that accordingly :)

**Comment by OlMue** [comment]
@tkowark @MarlisFriedl [#343](https://github.com/corona-warn-app/cwa-website/pull/343) was requested by the RKI today and probably will be merged earlier than this pull request. This might lead to a conflict, but can be resolved later when you're ready.

**Comment by mtb77** [comment]
Hi @OlMue, @MarlisFriedl is not responsible for the FAQs anymore, @svengabr and me are now in charge. 
Should we have a look at the conflicts? 

Cheers,
Sascha

**Comment by daimpi** [comment]
@mtb77 could you please revert https://github.com/corona-warn-app/cwa-website/pull/343 until we have clarification on this issue? B/c it currently introduces factually incorrect information into the FAQ: https://github.com/corona-warn-app/cwa-website/issues/307#issuecomment-684774771

**Comment by mtb77** [comment]
@daimpi let me clarify that first

**Comment by mtb77** [comment]
@daimpi Just had a call with Oliver. I will need to get back with both pull requests to @janetback and the RKI and will give you feedback shortly. Until then we will keep the PR merged.

Cheers,
Sascha

**Comment by daimpi** [comment]
@mtb77 
While I'm not particularly happy with leaving a PR that introduces factually wrong information into the [live FAQ](https://www.coronawarn.app/en/faq/#encounter_but_green) lingering, it will probably not cause too much problems if we can get this sorted out quickly.

Thanks for your effort 👍 🙂.

**Comment by daimpi** [comment]
@mtb77 any update on how you plan to proceed with this issue?

**Comment by mtb77** [comment]
@daimpi hi I will give you an update on this issue tomorrow - sorry for the delay!

Cheers,
Sascha

**Comment by mtb77** [comment]
@daimpi i merged master back into this branch - can you please doublecheck if i missed a change?

File: src/data/faq.json [hunk 2]
```diff
@@ -58,9 +58,10 @@
...
+                            "In the next step, the app analyzes all the matching rolling proximity identifiers for each diagnosis key, to estimate how long the exposure lasted in total on the day in question and how close the smartphones were to each other on average during the exposure. The distance is calculated from the measured reduction in strength of the Bluetooth signal, which is specified in dB (decibel). All exposures for a diagnosis key that lasted less than 10 minutes in total (regardless of how close the smartphones came during that time) or during which the smartphones were more than 8 meters (73 dB attenuation) apart on average (regardless of how long the exposure lasted) are rated as harmless. This is why the Corona-Warn-App can display encounters, but the risk status stays the same.",
+                            "The remaining encounters are rated collectively according to their distance and duration as well as the presumed infectiousness of the other person(s). This rating doesn't always yield an increased risk. This is why the Corona-Warn-App can display encounters, but the risk status stays the same.",
```
Comment (daimpi) [inline src/data/faq.json:63]: Nice!  
This change accurately describes the two ways how green encounters can happen 👍.
 
One thing I was wondering about, but unfortunately don't have a solution: The new entry now states twice in short succession 
> This is why the Corona-Warn-App can display encounters, but the risk status stays the same.

While this is o/c all correct, I was wondering whether we could formulate this a bit smoother, but I unfortunately don't have a suggestion right now. But it's also not really important, just some cosmetics. What do you think? 🙂

File: src/data/faq_de.json [hunk 3]
```diff
@@ -58,8 +58,8 @@
...
                             "Die verbleibenden Begegnungen werden in ihrer Gesamtheit unter Berücksichtigung ihres Abstands und ihrer Dauer sowie der vermutlichen Infektiosität der anderen Person(en) bewertet. Nicht immer ergibt diese Bewertung ein erhöhtes Risiko. Deswegen kann die Corona-Warn-App Ihnen Begegnungen gemeldet haben, ohne dass die Risikobewertung sich ändert.",
```
Comment (daimpi) [inline src/data/faq_de.json:63]: the English version states: 
>This is why the Corona-Warn-App can display encounters, but the risk status stays the same.

while the German one says:
>Deswegen kann die Corona-Warn-App Ihnen Begegnungen gemeldet haben, ohne dass die Risikobewertung sich ändert.

display encounters != Begegnung gemeldet

maybe the German version could be brought in line with the English one? Something like "Deswegen kann die Corona-Warn-App Ihnen Begegnungen anzeigen, […]"?

If you compare this (line 63) with the textblock above in line 62, there it has already been adjusted :).
It's not that important, just thought I'd mention it 🙂

**Comment by daimpi** [comment]
@mtb77 looks good to me, thanks 👍.

I've left two small remarks in the diff, but nothing important. 
So imho this can also just be merged as is 🙂.

**Review by mtb77 — APPROVED** [review]

File: src/data/faq.json [hunk 4]
```diff
@@ -59,7 +59,7 @@
...
+                            "In the next step, the app analyzes all the matching rolling proximity identifiers for each diagnosis key, to estimate how long the exposure lasted in total on the day in question and how close the smartphones were to each other on average during the exposure. The distance is calculated from the measured reduction in strength of the Bluetooth signal, which is specified in dB (decibel). All exposures for a diagnosis key that lasted less than 10 minutes in total (regardless of how close the smartphones came during that time) or during which the smartphones were more than 8 meters (73 dB attenuation) apart on average (regardless of how long the exposure lasted) are rated as harmless.",
```
[the remark this answers is not in the document]
Comment (mtb77) [reply src/data/faq.json:62]: I was struggling with the build :) yes please!

**Pull request merged**